plugin.audio.weareone/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.weareone" name="WeAreOne" version="1.0.0" provider-name="fossdd">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="script.module.routing" version="0.2.0"/>
+
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="main.py">
+        <provides>audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Play Audio Streams by the WeAreOne Family</summary>
+        <description lang="en_GB">WeAreOne is a radio sender family. It contains popular techno and house radios like TechnoBase.FM and HouseTime.FM.</description>
+        <language></language>
+        <platform>all</platform>
+        <license>unlicense</license>
+        <forum></forum>
+        <website></website>
+        <email>fossdd@tutanota.com</email>
+        <source></source>
+        <news></news>
```

Comment:
Drop if empty

Reply:
should the new commit also have the same name: `[plugin.audio.weareone] v1.0.0`?